### DISCUSSION TITLE
When opening Hogwarts Legacy (moonlight->sunshine), nothing happens

### DISCUSSION THREAD

**Discussion by shackrock** [LizardByte/community#170] [Sunshine] [answered]
I'm having an issue w/ trying to open hogwarts legacy from moonlight, on a pc running sunshine.

The connection comes up and it shows my desktop. The file itself (hogwarts) never opens.  When I head upstairs to use my mouse and keyboard to do it, I get this popup box, and I suspect this is what is causing the issue:

![image](https://user-images.githubusercontent.com/559901/218455194-d07e06ee-5de1-43e3-8397-768bd4852113.png)

My settings on the application in sunshine are pasted below:

![image](https://user-images.githubusercontent.com/559901/218455262-3ec16dcf-88ce-4be1-af34-8eb51abf0c86.png)

any tips?

**Comment by zachsheffler** [comment]
Try running the executable below rather than in the base directory:

```[...]\steamapps\common\Hogwarts Legacy\Phoenix\Binaries\Win64\HogwartsLegacy.exe```

**Comment by shackrock** [comment]
No luck still. With this, just nothing happens when I open from moonlight.  It's pretty odd.  It just opens to the desktop then I need to go to my PC mouse and open it manually.

**Comment by xoniq** [comment, marked as answer]
This is what works for me, detached command:

steam steam://rungameid/990080

One issue tho, when you exit the game, it cannot detect it closing because it's not hooked to the .exe, so it keeps the desktop open instead of closing to the moonlight menu.

**Reply by shackrock** [reply]
Alright, will give this one a try!
